### PR TITLE
Adding Gluon service

### PR DESCRIPTION
**Pull request by gkunz**
Signed-off-by: Georg Kunz <georg.kunz@ericsson.com>

### CODE REVIEW

File: puppet/services/gluon.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    value:
+      service_name: gluon
+      monitoring_subscription: {get_param: MonitoringSubscriptionGluon}
+      config_settings:
+        tripleo.gluon.firewall_rules:
+          '142 gluon':
+            dport:
+              - 2705
+      step_config: |
+        include tripleo::profile::base::gluon
```
Comment (trozet) [inline puppet/services/gluon.yaml]: Missing new line

**Review by trozet — COMMENTED** [review]
A couple things missing:
network/servicee_net_map.j2.yaml, need any entry for GluonNetwork: internal_api

Also, do you need to add a config to bind gluon to a specific IP? Like:
puppet/services/neutron-api.yaml:            neutron::bind_host: {get_param: [ServiceNetMap, NeutronApiNetwork]}

Also, does gluon need to be in keystone? If so, look at network/endpoints/endpoint_map.yaml, and we will also need to add some code to generate a password

**Comment by trozet** [comment]
One other thing, do we need gluon to be in HA? if so we will need to configure haproxy.

**Comment by gkunz** [comment]
Thanks for the hints. Currently Gluon is noha only, so no configuration of haproxy needed. Also, Gluon binds just to localhost on the controller and is being configured by the cli running on the controller as well. So, not need to bind to a specific IP for now. Integration in Keystone is on the roadmap for the next release.

**Pull request merged**